tests/grid/tria_clear_user_data.output

```diff
@@ -0,0 +1,4 @@
+
+DEAL::Cleared user data in 1D
+DEAL::Cleared user data in 2D
+DEAL::Cleared user data in 3D
```

Comment:
Please add an endline character to the end of this file, as well as to the changelog entry.

Reply:
done, thanks.